content/en/docs/devops-user-guide/how-to-use/jenkins-email.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Set Email Server for KubeSphere Pipeline"
+keywords: 'kubesphere, kubernetes, notification, jenkins, devops, ci/cd, pipeline'
+description: 'Set Email Server for KubeSphere CI/CD pipeline'
+---
+
+
+The built-in Jenkins cannot share the same email configuration with platform notification system. Thus we need to set email server for KubeSphere DevOps pipeline separately.
+
+> Note: Please be aware that the modification of the email server in `ks-jenkins` deployment below will restart the deployment itself. Consequently, the DevOps system will be unavailable for a few minutes. Please make such modification at an appropriate time.
+
+1. Log in KubeSphere by using `admin` account, navigate to **System-workspace → Projects → kubesphere-devops-system**.
```

Comment:
This step should be updated and synchronized to v3.0.0, see the preview page
https://deploy-preview-367--kubesphere-v3.netlify.app/docs/devops-user-guide/how-to-use/jenkins-email/

Reply:
fixed